.eslintrc.js

```diff
@@ -7,7 +7,9 @@ module.exports = {
   },
   extends: [
     'airbnb',
-    'plugin:@typescript-eslint/recommended',
+    "eslint:recommended",
+    "plugin:@typescript-eslint/eslint-recommended",
```

Comment:
What's the difference between typescript-eslint/eslint-recommended and typescript-eslint/recommended? I really can't get it.